website/docs/plugins/ray_launcher.md

```diff
@@ -30,21 +30,23 @@ defaults:
  `~/hydra/plugins/hydra_ray_launcher/hydra_plugins/hydra_ray_launcher/conf/hydra/launcher/ray_local.yaml`
  
 <!-- Add example link once it is available on hydra master -->
-An example using this launcher is provided in the plugin repository.
-
-Starting the app with `python my_app.py hydra/launcher=ray_local db=mysql,postgresql -m` will launch two parallel executions:
+An example using the ray local launcher is provided in the plugin repository.
 
 ```text
-$ python my_app.py hydra/launcher=ray_local task=1,2,3 -m 
-[HYDRA] Ray Launcher is launching 2 jobs
-[HYDRA] Sweep output dir : multirun/2020-04-20/14-40-36
-[HYDRA] Ray not initialized, init with config: {}
- INFO resource_spec.py:212 -- Starting Ray with 9.42 GiB memory available for workers and up to 4.73 GiB for objects. You can adjust these settings with ray.init(memory=<bytes>, object_store_memory=<bytes>).
- INFO services.py:1148 -- View the Ray dashboard at localhost:8265
-[HYDRA]        #0 : db=mysql
-[HYDRA]        #1 : db=postgresql
-(pid=24260) [__main__][INFO] - Hello world from my app with config: {'db': 'postgresql'}
-(pid=24261) [INFO] - Hello world from my app with config: {'db': 'mysql'}
+$ pwd
+<project_root>/hydra_ray_launcher/example/local
```

Comment:
As I said, I think we should use the same example with different configs for local and aws.

Reply:
fixed the example app